src/fontra/backends/ufo_utils.py

```diff
@@ -7,10 +7,10 @@
 
 
 _glyphNamePat = re.compile(rb'<glyph\s+name\s*=\s*"([^"]+)"')
-_unicodePat = re.compile(rb'<unicode\s+hex\s*=\s*"([^"]+)"')
+_codePointPat = re.compile(rb'<unicode\s+hex\s*=\s*"([^"]+)"')
```

Comment:
Suggest to revert this rename: this regex literally matches `<unicode ...` so the original name makes some sense.